Px.Utils/PxFile/PxFileSyntaxConf.cs

```diff
@@ -32,23 +32,38 @@ public class ValueSymbols
                 private const char TIME_SERIES_INTERVAL_START_DEFAULT = '(';
                 private const char TIME_SERIES_INTERVAL_END_DEFAULT = ')';
                 private const char TIME_SERIES_LIMITS_SEPARATOR_DEFAULT = '-';
+                private const string DATETIME_FORMAT_DEFAULT = "yyyyMMdd HH:mm";
 
                 public char StringDelimeter { get; set; } = STRING_DELIMETER_DEFAULT;
                 public char ListSeparator { get; set; } = LIST_SEPARATOR_DEFAULT;
                 public char TimeSeriesIntervalStart { get; set; } = TIME_SERIES_INTERVAL_START_DEFAULT;
                 public char TimeSeriesIntervalEnd { get; set; } = TIME_SERIES_INTERVAL_END_DEFAULT;
                 public char TimeSeriesLimitsSeparator { get; set; } = TIME_SERIES_LIMITS_SEPARATOR_DEFAULT;
+                public string DateTimeFormat { get; set; } = DATETIME_FORMAT_DEFAULT;
 
                 private ValueSymbols() { }
 
                 public static ValueSymbols DefaultValueSymbols => new();
             }
 
             private const char KEYWORD_SEPARATOR_DEFAULT = '=';
-            private const char SECTION_SEPARATOR_DEFAULT = ';';
+            private const char ENTRY_SEPARATOR_DEFAULT = ';';
+            private const char LINE_BREAK_DEFAULT = '\n';
+            private const char END_OF_STREAM_DEFAULT = '\0';
+            private const int SPACE = 0x20;
+            private const int HORIZONTAL_TAB = 0x09;
+            private const int CARRIAGE_RETURN = 0x0D;
+            private const int LINE_FEED = 0x0A;
 
             public char KeywordSeparator { get; set; } = KEYWORD_SEPARATOR_DEFAULT;
-            public char SectionSeparator { get; set; } = SECTION_SEPARATOR_DEFAULT;
+            public char EntrySeparator { get; set; } = ENTRY_SEPARATOR_DEFAULT;
+            public char Linebreak { get; set; } = LINE_BREAK_DEFAULT;
+            public char EndOfStream { get; set; } = END_OF_STREAM_DEFAULT;
+            public char Space { get; set; } = (char)SPACE;
```

Comment:
Fixed characters should not be editable properties. Move them is separate char constants collection or something.